tests/testthat/test_civis_ml_plot.R

```diff
@@ -3,7 +3,7 @@ context("civis_ml_plot")
 
 model_list    <- readRDS("data/civis_ml_models.rds")
 plotable_mods <- model_list[1:10]
-err_mod       <- model_list[11:12]
+err_mod       <- model_list[11:14]
```

Comment:
The new models are in a binary file, so I can't see them. I assume that the new error models are related to the no validation step?